docker/.env

```diff
@@ -8,6 +8,7 @@ MF_NGINX_MQTTS_PORT=8883
 
 # Message Broker
 MF_BROKER_TYPE=nats
+MF_BROKER_URL=nats://broker:${MF_NATS_PORT}
```

Comment:
We should probably remove `MF_NATS_URL` as @manuio mentioned. Also, we should set `MF_NATS_PORT=4222` **before** `MF_BROKER_URL=nats://broker:${MF_NATS_PORT}`.